derive_builder_core/src/setter.rs

```diff
@@ -117,6 +119,27 @@ impl<'a> ToTokens for Setter<'a> {
                     new.#field_ident = #option::Some(#into_value);
                     new
             }));
+            
+            if self.try_enabled {
+                let try_into = self.bindings.try_into_trait();
+                let try_ty_params = quote!(<VALUE: #try_into<#ty>>);
+                let try_ident = syn::Ident::new(format!("try_{}", ident));
+                let result = self.bindings.result_ty();
+                
+                tokens.append(quote!(
+                    #(#attrs)*
+                    #vis fn #try_ident #try_ty_params (#self_param, value: VALUE)
+                        -> #result<#return_ty, VALUE::Err>
+                    {
+                        #deprecation_notes
```

Comment:
Please remove this line, because we want to emit this only _once_ per setter combination.
Since we piggy back the try-variant onto the standard setter, it will already be emitted in the standard setter.